src/ui-components/CodapFlowSelect.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from "react";
+
+interface CodapFlowSelectProps<T extends string | number> {
+  onChange: React.ChangeEventHandler<HTMLSelectElement>;
+  defaultValue: T;
+  value: T | null;
+  options: {
+    value: T;
+    title: string;
+  }[];
+}
+
+export default function CodapFlowSelect<T extends string | number>({
+  onChange,
+  value,
+  defaultValue,
+  options,
+}: CodapFlowSelectProps<T>): ReactElement {
+  return (
+    <select
+      onChange={onChange}
+      defaultValue={defaultValue}
+      value={value || defaultValue}
```

Comment:
What's the intention here with `value || defaultValue`? This seems to cause an error in the console, telling us to define either `defaultValue` or `value` but not both. I think maybe it's fine to just have `defaultValue` here?